android/assets/jsons/translations/German.properties

```diff
@@ -631,7 +629,7 @@ Hurry Research = Forschung beschleunigen
 Conduct Trade Mission = Handelsmission durchführen
 Your trade mission to [civName] has earned you [goldAmount] gold and [influenceAmount] influence! = Deine Handelsmission zu [civName] hat dir [goldAmount] Gold und [influenceAmount] Einfluss eingebracht!
 Hurry Wonder = Wunder beschleunigen
-Your citizens have been happy with your rule for so long that the empire enters a Golden Age! = Deine Bürger sind so zufrieden mit deiner Herrschaft, dass dein Imperium in ein Goldenes Zeitalter eintritt!
+Your citizens have been happy with your rule for so long that the empire enters a Golden Age! = Deine Bürger sind so zufrieden mit deiner Herrschaft, dass dein Reich in ein Goldenes Zeitalter eintritt!
```

Comment:
Ob wir damit unangenehm auffallen, "Deutsch" und "Reich"?? OK "Imperium" durch die Bank ist holperig und nicht so von der Klangfarbe wie englisch 'nation'... "Nation"? Oder "Imperium" war ein versteckter Witz den keiner versteht... Ich kann mich partout nicht daran erinnern, ob das Imperium von mir ist... Vielleicht hatte das was mit dem Datumsformat was ja in Unciv ein inkonsistentes Problem ist zu tun... (Java 6 Datumstypen Katastrophe, besser in 8 weil ein Projekt namens... *** übernommen wurde - aber Unciv soll ja alte Androiden unterstützen - jedenfalls wäre da eine Assoziation zu dem SciFi-Imperium...)

Reply:
Was hat denn Imperium mit einem Datumstyp zu tun? xD
Hatte wiedermal die Konsistenz im Sinn und da bisher "Reich" öfter vorkam als "Imperium", habe ich es gleich gezogen.
Bei Imperium muss ich immer an die SciFi Filme denken. Daher passt es für mich im Kontext mit, mehr oder weniger genauen, weltgeschichtlichen Ereignissen nicht so richtig.
Ich glaube und hoffe, dass das Repo nicht plötzlich gesperrt wird, nur weil jetzt 52 mal "reich" in der deutschen Übersetzungsdatei vorkommt. Wäre etwas lächerlich. Immerhin is es ein normaler deutscher Begriff.